plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -124,6 +125,7 @@
     private static final JsonCodec<NodesResponse> NODES_RESPONSE_CODEC = jsonCodec(NodesResponse.class);
     private static final JsonCodec<CountResponse> COUNT_RESPONSE_CODEC = jsonCodec(CountResponse.class);
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapperProvider().get();
+    private static final Set<String> DATA_ROLES = ImmutableSet.of("data", "data_hot", "data_content");
```

Comment:
Let's also add `data_warm` and `data_cold` here.

Reply:
done, but please accept it  finally  , we spent too much time to try  a feature that we are not sure to adopt and we have to decide soon